tests/test-sources/plugins/utils/guess-indent.nix

```diff
@@ -0,0 +1,17 @@
+{
+  empty = {
+    plugins.guess-indent.enable = true;
+  };
+
```

Comment:
Could you please add a `defaults` test case, with all the options explicitly set to their default value ?

Reply:
Isn't that just what `empty` does? Or should I explicitly set all options to their default value in another testcase?

Reply:
No, `empty` will leave all options set to `null` (which we will filter out when calling `setup`).

The "plugin defaults" exist only as documentation, so a `defaults` test is to check they actually work when explicitly set.

(There's no way to test they are actually the default upstream though, afaik)